app/dashboard/views.py

```diff
@@ -776,7 +776,9 @@ def users_fetch(request):
     else:
         network = 'rinkeby'
 
-    user_list = Profile.objects.prefetch_related('fulfilled', 'leaderboard_ranks', 'feedbacks_got').order_by(order_by)
+    user_list = Profile.objects.prefetch_related(
```

Comment:
Can we call this `profile_list` instead since there is already a `User` model?

Reply:
true maybe in plural? `profiles_list` ?

Reply:
I like `profile_list` personally but either one is ok 👍